app/views/idv/hybrid_mobile/document_capture/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render(
+      'idv/shared/document_capture',
+      document_capture_session_uuid: document_capture_session_uuid,
+      flow_path: 'hybrid',
+      sp_name: decorated_session.sp_name,
+      failure_to_proof_url: idv_doc_auth_return_to_sp_url,
```

Comment:
Another recent change in the desktop DocumentCapture code, make this take a `failure_to_proof_url` variable that gets [set directly](https://github.com/18F/identity-idp/blob/0a1bd4209f6c0d9ee34e698ad3fc63ee01a87ec7/app/controllers/idv/document_capture_controller.rb#L45) rather than taking a detour through a route to DocAuthController#return_to_sp.

Reply:
Addressed in [9624023](https://github.com/18F/identity-idp/pull/8243/commits/9624023bb36729e5842d3c702b97ddcfd6972cf2)

Reply:
It's used [in the shared template](https://github.com/18F/identity-idp/blob/ed495cb006a3038d7a8d81854078bdb97861f3dd/app/views/idv/shared/_document_capture.html.erb#L40). It's reachable when there's a barcode error or something like that, which I think could happen on mobile too.